content/en/agent/guide/upgrade_between_minor_versions.md

```diff
@@ -45,3 +66,89 @@ Upgrading to the latest Agent minor version:
 
 {{% /tab %}}
 {{< /tabs >}}
+
+
+## Upgrade between major versions of the Agent
+### Upgrade to Datadog Agent 7 
```

Comment:
The hierarchy of these headings seems unnecessarily complicated. Can we just have two headings, "upgrade from 6 to 7" and "upgrade from 5 to 7"?